packages/mobile/lib/auxiliary/drawer/actions/closeDrawer.action.ts

```diff
@@ -0,0 +1,6 @@
+import { DrawerId } from '../enums'
+import { drawersStore } from '../stores'
+
+export function closeDrawer(drawerId: DrawerId): void {
+    drawersStore.update(($drawersStore) => $drawersStore.filter((drawer) => drawer.id !== drawerId))
```

Comment:
Might be good to extract this line into a `removeDrawerState` function inside the store file, then call it here. It might prevent logic from being duplicated later if we need to remove a drawer state for some reason.

Reply:
Ah and we also don't need `.action` in the filename - for actions we use `camelCase` (reason is that's just what we started doing initially). If using `kebab-case` then we should be specifying the file type in the filename.